scipy/ndimage/tests/test_ndimage.py

```diff
@@ -4481,6 +4482,8 @@ def test_white_tophat02(self):
                                       structure=structure)
         assert_array_almost_equal(expected, output)
 
+    @pytest.mark.xfail(platform.machine() == 'ppc64le',
+		       reason="see gh-9515")
```

Comment:
there's already an issue open for this one, but reviewers may need to decide which other test failures merit their own issues